src/Core/CommandLine.cs

```diff
@@ -32,8 +32,8 @@ public static async Task<int> RunApp(Config config)
         {
             NgsaLog logger = new NgsaLog { Name = typeof(App).FullName };
 
-            // the first read is always 0
-            _ = PerfCounters.GetCpu();
+            // start collecting CPU usage
+            CpuCounter.Start();
```

Comment:
Should we add CpuCounter.Stop() after the `await w.ConfigureAwait(false);` line?

Reply:
I added IDisposable to the class